emission/tests/modellingTests/TestGreedySimilarityBinning.py

```diff
@@ -95,33 +104,17 @@ def testNoPrediction(self):
             "purpose_confirm": ['pizza_party'],
             "replaced_mode": ['crabwalking']
         }
-
         n = 5
-        train = etmm.generate_mock_trips(
-            user_id="joe", 
-            trips=n, 
-            origin=(39.7645187, -104.9951944),       # Denver, CO
-            destination=(39.7435206, -105.2369292),  # Golden, CO
-            label_data=label_data, 
-            threshold=0.001,  # ~ 111 meters in degrees WGS84
+
+        train = etmu.setTripConfig(trips=n, org=(39.7645187, -104.9951944), # Denver, CO
+                                   dest=(39.7435206, -105.2369292),  # Golden, CO
+                                   trip_part='od', label_data=label_data                                 
         )
-        test = etmm.generate_mock_trips(
-            user_id="joe", 
-            trips=1, 
-            origin=(61.1042262, -150.5611644),       # Anchorage, AK
-            destination=(62.2721466, -150.3233046),  # Talkeetna, AK
-            label_data=label_data, 
-            threshold=0.001,  # ~ 111 meters in degrees WGS84
+        test = etmu.setTripConfig(trips=n, org=(61.1042262, -150.5611644), # Denver, CO
+                                   dest=(62.2721466, -150.3233046),  # Golden, CO
+                                   trip_part='od', label_data=label_data,                                   
         )
-
-        model_config = {
-            "metric": "od_similarity",
-            "similarity_threshold_meters": 500,      # meters,
-            "apply_cutoff": False,
-            "incremental_evaluation": False
-        }
-        model = eamtg.GreedySimilarityBinning(model_config)
-
+        model = etmu.setModelConfig("od_similarity",  500,  False, "origin-destination", False)
```

Comment:
I am not sure that this is actually more clean that just creating the model config. I was hoping that you would refactor the code around creating multiple configs, one for o, one for d, and one for od

Reply:
Here (in functions `testNoPrediction` and `testPrediction`), I replaced this  just to make the trip setups and model calls uniform throughout this file ,i.e., everytime a trip is setup, it uses `setTripConfig` and every time binning is called using `setModelConfig`.  

The multiple configs are not required in functions `testNoPrediction` and `testPrediction`. 
And, where multiple configs are need  in `testNoBinning` and `testBinning` , this is handled through parametrized subTests.